### PR TITLE
Remove S3 support and cleanup args

### PR DESCRIPTION
**Pull request by tysmith**

### CODE REVIEW

File: grizzly/args.py [hunk 1]
```diff
@@ -275,9 +273,8 @@ def sanity_check(self, args):
         if args.timeout is not None and args.timeout < 1:
             self.parser.error("--timeout must be >= 1")
 
-        if "tool" not in self._sanity_skip:
-            if args.tool is not None and not args.fuzzmanager:
-                self.parser.error("--tool can only be given with --fuzzmanager")
+        if args.tool and not args.fuzzmanager:
```
Comment (jschwartzentruber) [inline grizzly/args.py:276]: Should be `args.tool is not None`. This will allow `--tool ""` without `--fuzzmanager`.

**Review by jschwartzentruber — APPROVED** [review]
Very nice cleanup.

File: grizzly/args.py (hunk 1 above)
Comment (tysmith) [reply grizzly/args.py:276]: --tool requires an argument otherwise you get `error: argument --tool: expected one argument`

Comment (jschwartzentruber) [reply grizzly/args.py:276]: I didn't say no argument, I said empty argument.

```
$ python -m grizzly.replay --tool "" ~/builds/asan/firefox testcase.html
[2022-10-14 13:09:41] Starting Grizzly Replay
[2022-10-14 13:09:41] Ignoring: log-limit, timeout
[2022-10-14 13:09:41] Using time limit: 30s, timeout: 45s
[2022-10-14 13:09:41] Repeat: 1, Minimum crashes: 1, Relaunch 1
^C[2022-10-14 13:09:42] Shutting down...
[2022-10-14 13:09:42] Done.
```

Comment (tysmith) [reply grizzly/args.py:276]: Oh I didn't know that was possible. OK then.

**Pull request merged**